Introspect/ViewExtensions.swift

```diff
@@ -11,6 +11,10 @@ extension View {
     public func inject<SomeView>(_ view: SomeView) -> some View where SomeView: View {
         return overlay(view.frame(width: 0, height: 0))
     }
+    
+    public func inject2<SomeView>(_ view: SomeView) -> some View where SomeView: View {
+        return background(view.frame(width: 0, height: 0))
+    }
```

Comment:
I don't think this is the best way to go about this. The name `inject2` is not very descriptive and I don't think we should add another method here. Since `inject` is part of the public API we should not add cryptic methods to this. Keep in mind that many people extend Introspect themselves.

Imo `inject` should be updated to take an `enum InjectionSite` or something like that, with a value of `overlay` or `background`.

Since this method is public API and to prevent any breakage, the new parameter should have a default value of `overlay` and come after the `view` parameter. The new `background` behavior can then be specifically targeted by introspection methods that depend on it.

Reply:
This was a mistake, I used this as a test and didn't remove it. It's not needed.

Reply:
Removed.